projects/addon-editor/components/editor/editor.component.ts

```diff
@@ -108,12 +108,21 @@ export class TuiEditorComponent
     }
 
     override writeValue(value: string | null): void {
+        if (value === this.value) {
```

Comment:
Shouldn't this check be after contentProcessor?

Reply:
Markdown for some reason when parsing it can substitute some tags recursively,and because of this our content can always be different from the previous one

1) `contentProcessor('')` -> `<p></p>` -> `setValue($)`
2) `setValue` again trigger `writeValue` -> `setValue`
3) `contentProcessor('<p></p>')` -> `<p></p><p></p>` -> `setValue($)`
4) `setValue` again trigger `writeValue` -> `setValue`
5) `contentProcessor('<p></p><p></p>')` -> `<p></p><p></p><p></p>` -> `setValue($)`
N) ...